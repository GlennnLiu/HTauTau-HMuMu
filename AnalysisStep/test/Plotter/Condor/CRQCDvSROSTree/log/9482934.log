000 (9482934.004.000) 07/27 15:05:28 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_8624_ecd7>
...
040 (9482934.004.000) 07/27 15:06:58 Started transferring input files
	Transferring to host: <128.142.61.95:9618?addrs=128.142.61.95-9618+[2001-1458-301-23--100-5e]-9618&alias=b7s06p7589.cern.ch&noUDP&sock=slot1_17_4941_21a2_158606>
...
040 (9482934.004.000) 07/27 15:06:58 Finished transferring input files
...
001 (9482934.004.000) 07/27 15:06:59 Job executing on host: <128.142.61.95:9618?addrs=128.142.61.95-9618+[2001-1458-301-23--100-5e]-9618&alias=b7s06p7589.cern.ch&noUDP&sock=startd_4496_75e4>
...
006 (9482934.004.000) 07/27 15:07:08 Image size of job updated: 172952
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
040 (9482934.004.000) 07/27 15:07:08 Started transferring output files
...
040 (9482934.004.000) 07/27 15:07:08 Finished transferring output files
...
005 (9482934.004.000) 07/27 15:07:08 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:03, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:03, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	24298  -  Run Bytes Sent By Job
	352  -  Run Bytes Received By Job
	24298  -  Total Bytes Sent By Job
	352  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       37        1   1275694 
	   Memory (MB)          :        0     2000      2000 

	Job terminated of its own accord at 2023-07-27T13:07:08Z.
...
