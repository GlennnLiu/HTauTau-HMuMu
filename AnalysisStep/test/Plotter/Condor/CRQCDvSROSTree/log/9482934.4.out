[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.diBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.triBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.top.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.EWKZee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.EWKZtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.EWKZmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.DYee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.DYtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.DYmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.Wjet.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.TTfake.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.Data.root
[INFO] Total MC summed
[INFO] Saved Histos to /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDvSROSTree.TotalMC.root
